### PR TITLE
fix gemspec

### PR DESCRIPTION
**Pull request by kirillshevch**

### CODE REVIEW

File: vertigo-rtm.gemspec [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
   s.version     = Vertigo::Rtm::VERSION
-  s.authors     = ["chubchenko"]
-  s.email       = ["artem.chubchenko@gmail.com"]
-  s.homepage    = "TODO"
-  s.summary     = "TODO: Summary of Vertigo::Rtm."
-  s.description = "TODO: Description of Vertigo::Rtm."
-  s.license     = "MIT"
+  s.authors     = ['chubchenko']
+  s.email       = ['artem.chubchenko@gmail.com']
+  s.homepage    = 'https://rubygarage.org'
```
Comment (vshmyhlo) [inline vertigo-rtm.gemspec]: i suggest to set homepage to github repo page, what do you think?

**Comment by babakhanov** [comment]
@v-shmyhlo fixed

**Comment by kirillshevch** [comment]
@v-shmyhlo fixed

**Comment by chubchenko** [comment]
@kirillweb can you reopen this PR to develop branch

**Pull request closed without merging**